install/build_synthea_db.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Start Docker containers
+echo "Starting containers..."
+sudo docker start pg
+sudo docker start synthea
+
+# Delete existing database
+sudo docker exec -it pg psql -U postgres -c "DROP DATABASE phit WITH (FORCE);"
+
+# Create PHIT database and Synthea schema
+echo "Creating Synthea database..."
+sudo docker exec -it pg psql -U postgres -c "CREATE DATABASE phit;"
+sudo docker exec -it pg psql -U postgres -d phit -c "CREATE SCHEMA IF NOT EXISTS synthea;"
+
+# Create relation structure for database
+echo "Creating relation structure for database..."
+sudo docker cp /home/ubuntu/predictive-healthcare-infrastructure-tool/install/sql/synthea_template.sql pg:/home
+sudo docker exec -it pg psql -U postgres -d phit -f "/home/synthea_template.sql"
+
+# Generate Synthea Data
+sudo docker exec -it synthea bash -c "java -jar synthea-with-dependencies.jar -p 1000 --exporter.csv.export true"
```

Comment:
In the future, we may want to pass the populationSize parameter as an environmental variable. For the system's initial back-end, this is sufficient.